server/server.js

```diff
@@ -11,7 +12,10 @@ app.get('/listing/:listingId', (req, res) => {
 
 app.get('/listing/:listingId/booked', (req, res) => {
   // returns array of dates that are booked/unavailable
-  res.end();
+  Listing.find({listingId: req.params.listingId}, (err, listings) => {
+    if (err) console.log(err);
```

Comment:
I think it is best practice if you use an if and else statement here. Also, instead of just console logging the error I suggest that you send the error message instead. Doing both these things will handle the error better.

Reply:
Done!